File: Weekly Assignments/Practical6/MatchingProfilesA.py
sus=[]
cri=[]
#Taking input from the user in string and spliting them by space
sus=input("Enter the 10 DNA of the suspect: ").split(" ")
cri=input("Enter the 10 DNA of the criminal: ").split(" ")

match = True
for i in range (10):
    #checks if each dna is same or not
    if sus[i]!=cri[i]:
         match=False
         #breaks if any one chromosome doesnot match
         break

if match == True:
    print("Repeated offender: Two profile matches")

else:
    print("Profile doesnot match")

File: Weekly Assignments/extra/matchingProfilesA.c
#include <stdio.h>
#include <stdbool.h>

int main()
{
    int size =10;
    float suspect [size];
    float criminal [size];

    printf("Enter the 10 chromosomes of the suspect separated by spaces: \n");
    for(int i=0; i<size; i++)
    {
        scanf("%f",&suspect[i]);
    }

    printf("Enter the 10 chromosomes of the criminal separated by spaces: \n");
    for(int i=0; i<size; i++)
    {
        scanf("%f",&criminal[i]);
    }

    bool match = true;

    for(int i=0; i<size; i++)
    {
        if(suspect[i]!=criminal[i])
        {
            match = false;
        }
    }
    if(match==true)
    {
        printf("The two profile matches!!!\n");
    }
    else
    {
        printf("The two profile don't match! \n");
    }

    return 0;
}